src/sentry/static/sentry/app/routes.jsx

```diff
@@ -743,13 +741,16 @@ function routes() {
       <Route path="/:orgId/" component={errorHandler(OrganizationDetails)}>
         <Route component={errorHandler(OrganizationRoot)}>
           <IndexRoute component={errorHandler(OrganizationDashboard)} />
-
           <Route
             path="/organizations/:orgId/discover/"
             componentPromise={() =>
-              import(/*webpackChunkName:"OrganizationDiscover"*/ './views/organizationDiscover/index')}
+              import(/*webpackChunkName: OrganizationDiscover*/ './views/organizationDiscover/index')}
```

Comment:
This needs quotes around `OrganizationDiscover`

Reply:
Why does it need quotes? Many of them don't have it

Reply:
@lynnagara 	yeah it's wrong, I came upon this in webpack4 - here's a PR to fix the rest: https://github.com/getsentry/sentry/pull/10047